File: js/seed.js
/* ══════════════════════════════════════════════
   seed.js — daftar bawaan (bank, e-wallet, kategori)
   Ini SARAN, bukan data. Semua yang benar-benar
   dipakai diinput sendiri oleh pengguna.
   ══════════════════════════════════════════════ */

const SEED_AKUN = [
  { kode:'tunai',     nama:'Uang Tunai',       jenis:'tunai',   ini:'Rp' },

  { kode:'bca',       nama:'BCA',              jenis:'bank',    ini:'BCA' },
  { kode:'mandiri',   nama:'Mandiri',          jenis:'bank',    ini:'MDR' },
  { kode:'bri',       nama:'BRI',              jenis:'bank',    ini:'BRI' },
  { kode:'bni',       nama:'BNI',              jenis:'bank',    ini:'BNI' },
  { kode:'bsi',       nama:'BSI',              jenis:'bank',    ini:'BSI' },
  { kode:'cimb',      nama:'CIMB Niaga',       jenis:'bank',    ini:'CMB' },
  { kode:'permata',   nama:'Permata',          jenis:'bank',    ini:'PRM' },
  { kode:'danamon',   nama:'Danamon',          jenis:'bank',    ini:'DNM' },
  { kode:'btn',       nama:'BTN',              jenis:'bank',    ini:'BTN' },
  { kode:'ocbc',      nama:'OCBC',             jenis:'bank',    ini:'OCB' },
  { kode:'panin',     nama:'Panin',            jenis:'bank',    ini:'PNN' },
  { kode:'maybank',   nama:'Maybank',          jenis:'bank',    ini:'MYB' },
  { kode:'mega',      nama:'Bank Mega',        jenis:'bank',    ini:'MGA' },
  { kode:'btpn',      nama:'BTPN / Jenius',    jenis:'bank',    ini:'BTP' },
  { kode:'jago',      nama:'Bank Jago',        jenis:'bank',    ini:'JGO' },
  { kode:'seabank',   nama:'SeaBank',          jenis:'bank',    ini:'SEA' },
  { kode:'blu',       nama:'blu by BCA',       jenis:'bank',    ini:'BLU' },
  { kode:'allo',      nama:'Allo Bank',        jenis:'bank',    ini:'ALO' },
  { kode:'neo',       nama:'Bank Neo Commerce',jenis:'bank',    ini:'NEO' },
  { kode:'muamalat',  nama:'Muamalat',         jenis:'bank',    ini:'MUA' },

  { kode:'gopay',     nama:'GoPay',            jenis:'ewallet', ini:'GO' },
  { kode:'ovo',       nama:'OVO',              jenis:'ewallet', ini:'OVO' },
  { kode:'dana',      nama:'DANA',             jenis:'ewallet', ini:'DNA' },
  { kode:'shopeepay', nama:'ShopeePay',        jenis:'ewallet', ini:'SPY' },
  { kode:'linkaja',   nama:'LinkAja',          jenis:'ewallet', ini:'LNK' },
  { kode:'flip',      nama:'Flip',             jenis:'ewallet', ini:'FLP' },
  { kode:'astrapay',  nama:'AstraPay',         jenis:'ewallet', ini:'AST' },

  { kode:'lainnya',   nama:'Lainnya — tulis sendiri', jenis:'lainnya', ini:'···' }
];

const SEED_KATEGORI_KELUAR = [
  'Makan & Minum', 'Belanja Harian', 'Transport', 'Bensin', 'Pulsa & Internet',
  'Listrik & Air', 'Sewa / Kos', 'Kesehatan', 'Pendidikan', 'Hiburan',
  'Pakaian', 'Perawatan Diri', 'Donasi & Sedekah', 'Hadiah', 'Cicilan',
  'Servis & Perbaikan', 'Pajak & Admin', 'Lain-lain'
];

const SEED_KATEGORI_MASUK = [
  'Gaji', 'Bonus', 'Usaha', 'Freelance', 'Hadiah', 'Penjualan Barang',
  'Bunga & Investasi', 'Iuran Diterima', 'Pengembalian', 'Lain-lain'
];

const WARNA_KANTONG = ['#0e9f6e','#c2760a','#2563eb','#9333ea','#db2777','#0891b2','#65a30d','#dc2626'];

File: native/app/src/main/java/id/my/ibstudio/dompetq/data/Seed.kt
package id.my.ibstudio.dompetq.data

/* ══════════════════════════════════════════════
   Daftar bawaan — SARAN, bukan data. Semua yang
   benar-benar dipakai diinput sendiri oleh pengguna.
   Disalin dari js/seed.js agar dua versi seragam.
   ══════════════════════════════════════════════ */

data class SeedAkun(val kode: String, val nama: String, val jenis: String, val ini: String)

val SEED_AKUN = listOf(
    SeedAkun("tunai", "Uang Tunai", "tunai", "Rp"),

    SeedAkun("bca", "BCA", "bank", "BCA"),
    SeedAkun("mandiri", "Mandiri", "bank", "MDR"),
    SeedAkun("bri", "BRI", "bank", "BRI"),
    SeedAkun("bni", "BNI", "bank", "BNI"),
    SeedAkun("bsi", "BSI", "bank", "BSI"),
    SeedAkun("cimb", "CIMB Niaga", "bank", "CMB"),
    SeedAkun("permata", "Permata", "bank", "PRM"),
    SeedAkun("danamon", "Danamon", "bank", "DNM"),
    SeedAkun("btn", "BTN", "bank", "BTN"),
    SeedAkun("ocbc", "OCBC", "bank", "OCB"),
    SeedAkun("panin", "Panin", "bank", "PNN"),
    SeedAkun("maybank", "Maybank", "bank", "MYB"),
    SeedAkun("mega", "Bank Mega", "bank", "MGA"),
    SeedAkun("btpn", "BTPN / Jenius", "bank", "BTP"),
    SeedAkun("jago", "Bank Jago", "bank", "JGO"),
    SeedAkun("seabank", "SeaBank", "bank", "SEA"),
    SeedAkun("blu", "blu by BCA", "bank", "BLU"),
    SeedAkun("allo", "Allo Bank", "bank", "ALO"),
    SeedAkun("neo", "Bank Neo Commerce", "bank", "NEO"),
    SeedAkun("muamalat", "Muamalat", "bank", "MUA"),

    SeedAkun("gopay", "GoPay", "ewallet", "GO"),
    SeedAkun("ovo", "OVO", "ewallet", "OVO"),
    SeedAkun("dana", "DANA", "ewallet", "DNA"),
    SeedAkun("shopeepay", "ShopeePay", "ewallet", "SPY"),
    SeedAkun("linkaja", "LinkAja", "ewallet", "LNK"),
    SeedAkun("flip", "Flip", "ewallet", "FLP"),
    SeedAkun("astrapay", "AstraPay", "ewallet", "AST")
)

val SEED_KATEGORI_KELUAR = listOf(
    "Makan & Minum", "Belanja Harian", "Transport", "Bensin", "Pulsa & Internet",
    "Listrik & Air", "Sewa / Kos", "Kesehatan", "Pendidikan", "Hiburan",
    "Pakaian", "Perawatan Diri", "Donasi & Sedekah", "Hadiah", "Cicilan",
    "Servis & Perbaikan", "Pajak & Admin", "Lain-lain"
)

val SEED_KATEGORI_MASUK = listOf(
    "Gaji", "Bonus", "Usaha", "Freelance", "Hadiah", "Penjualan Barang",
    "Bunga & Investasi", "Iuran Diterima", "Pengembalian", "Lain-lain"
)

val WARNA_KANTONG = listOf(
    "#0e9f6e", "#c2760a", "#2563eb", "#9333ea",
    "#db2777", "#0891b2", "#65a30d", "#dc2626"
)

val HARI = listOf("Minggu", "Senin", "Selasa", "Rabu", "Kamis", "Jumat", "Sabtu")
